src/Components/Artwork/Metadata.tsx

```diff
@@ -36,16 +35,17 @@ export const Metadata: React.FC<MetadataProps> = ({
   showSaveButton,
   ...rest
 }) => {
-  const LinkContainer = disableRouterLinking ? DisabledLink : RouterLink
+  const LinkContainer = disableRouterLinking ? Box : RouterLink
 
   return (
+    // @ts-ignore
```

Comment:
Typescript complains because `Box` doesn't allow all props from `rest`. Any idea how to fix this?